components/BookshelfItem.vue

```diff
@@ -113,24 +118,40 @@ const menuItems = computed<DropdownMenuItem[]>(() => {
         break
     }
 
-    return {
+    contentItems.push({
       label,
       icon: 'i-material-symbols-book-5-outline',
-      onSelect: () => {
-        openContentURL(contentURL)
-      },
+      onSelect: () => openContentURL(contentURL),
+    })
+
+    const isContentURLsEncrypted = !!(
+      contentURL.url.startsWith(encryptedArweaveLinkEndpoint)
+      || contentURL.url.includes('?key=')
+    )
+    if (!isContentURLsEncrypted) {
```

Comment:
since this is under for each loop, no need to add s,
also need to accommodate `hideDownload` of store info
```suggestion
    const isContentURLEncrypted = !!(
      contentURL.url.startsWith(encryptedArweaveLinkEndpoint)
      || contentURL.url.includes('?key=')
    )
    if (bookInfo.isDownloadable.value && !isContentURLEncrypted) {
```

Reply:
[🐛 Update logic for handling download items](https://github.com/likecoin/3ook-com/pull/47/commits/55de6c3a90bd03139a8b75a2fd93135a2746618f)